java/client/src/test/java/glide/api/RedisClientTest.java

```diff
@@ -975,6 +976,32 @@ public void strlen_returns_success() {
         assertEquals(value, payload);
     }
 
+    @SneakyThrows
+    @Test
+    public void setrange_returns_success() {
+        // setup
+        String key = "testKey";
+        int offset = 42;
+        String str = "pewpew";
+        String[] arguments = new String[] {key, Integer.toString(offset), str};
+        Long value = 10L;
```

Comment:
minor: value -> responseValue (value is confusing in the key-value context)

Reply:
There are 99 tests in this file and almost all of them have variable named "value".
I'm going to refactor tests in another PR (and rename the var). Is it OK for you?

Reply:
`commandResult` or `futureResult` is even less ambiguous.